src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/rate-limits-nrql-queries.mdx

```diff
@@ -8,34 +8,28 @@ redirects:
 
 import queriesnrqlEventCountinQueryBuilder from 'images/queries-nrql_screenshot-crop_event-count-in-query-builder.webp'
 
-Our New Relic query language, NRQL, has rate limits in place to ensure a high level of availability and reliability for all users. 
+Our [New Relic query language](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language), NRQL, has rate limits in place to ensure a high level of availability and reliability for all our users. 
```

Comment:
@zuluecho9 links like this might lure readers to another page. What's the need for this link here? Don't we want them to focus on the task at hand, which is understanding limits--not learning about NRQL?